ban/core/models.py

```diff
@@ -169,7 +175,7 @@ def compute_cia(self):
         if not get_fantoir:
             return None
         return compute_cia(str(self.parent.municipality.insee),
-                           get_fantoir(), self.number, self.ordinal)
+                           get_fantoir, self.number, self.ordinal)
```

Comment:
That's not OK, `get_fantoir` is a callable.
As discussed in chat, you can totally remove the need for checking parent.get_fantoir: now the parent will always have this method.